hack/run-e2e.sh

```diff
@@ -27,19 +27,17 @@ fi
 
 if [ -z "$tag" ]; then
   echo "-------- tag is empty, assuming default from the Operator"
+else
+  E2E_PARAMS="${E2E_PARAMS} --image quay.io/kiegroup/kogito-cloud-operator:${tag}"
 fi
 
 echo "-------- Running e2e tests with namespace=${namespace}, tag=${tag}, native=${native} and maven_mirror=${maven_mirror}"
 
 # creates the namespace
 oc create namespace ${namespace}
-# gives permissions
```

Comment:
Why did you remove this? Without the required permissions the operator won't run correctly.

Reply:
Originally I thought that these permissions are handled by operator-sdk when running the tests. But as I check the documentation, the role.yaml and role_binding.yaml are not applied. I will return it back.